### PR TITLE
UI improvement + change UI filtering

### PR DESCRIPTION
**Pull request by sugarshin**
- [fix: filter ui by mini query param](https://github.com/line/liff-playground/pull/20/commits/ad7595b15ee50c1ab29a25650ea68e2d14aeb44c)
- [fix: show liff url](https://github.com/line/liff-playground/pull/20/commits/a1014a6303005666fd8db107e1e4f4b0d5161d9a)
- [feat: add qr code for liff url](https://github.com/line/liff-playground/pull/20/commits/cfa7f8b20f95629ef3c55b6d0acd94e9c9b3ce95)

internal ref: LIFFP-5075

## as-is

ss00

## to-be

ss01

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-import { FilterContext, FilterTypes, FilterType } from './Context'
+import { FilterContext, FilterTypes } from './Context'
+import qrCode from './qr-code.png'
 
-const getFilterOptions = () => {
-  return Object.entries(FilterTypes).map(([value, label]) => ({
-    label, value
-  }))
-}
+const isMINI = new URLSearchParams(location.search).has('mini')
```
Comment (sugarshin) [inline src/App.tsx:10]: we will use `?mini` query param to the playground for MINI

**Review by kazuma0129 — APPROVED** [review]

**Review by odanado — APPROVED** [review]
LGTM

**Comment by sugarshin** [comment]
hi, @LeeChSien could you please code review when you have time? thank you!

File: src/Context.tsx [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 export const FilterTypes = {
-  ALL: 'ALL',
   LIFF: 'LIFF',
   MINI: 'MINI'
 } as const
 
 export type FilterType = keyof typeof FilterTypes
 
-export const FilterContext = React.createContext<FilterType>(FilterTypes.ALL)
+export const FilterContext = React.createContext<FilterType>(FilterTypes.LIFF)
```
Comment (LeeChSien) [inline src/Context.tsx:10]: Thanks for replacing it with `Context`. 👍

File: src/App.tsx [hunk 3]
```diff
@@ -28,15 +24,9 @@ function App() {
...
             readonly
-            value={'LIFF ID: ' + import.meta.env.VITE_LIFF_ID || ''}
-          />
-          <Select
-            value={filter}
-            options={getFilterOptions()}
-            handleChange={(e) => {
-              setFilter(e.target.value as FilterType)
-            }}
+            value={'LIFF URL: https://liff.line.me/' + import.meta.env.VITE_LIFF_ID || ''}
```
Comment (LeeChSien) [inline src/App.tsx]: nit:
```suggestion
            value={`LIFF URL: https://liff.line.me/${import.meta.env.VITE_LIFF_ID.toString()}`}
```

**Review by LeeChSien — APPROVED** [review]
LGTM

File: src/App.tsx (hunk 3 above)
Comment (sugarshin) [reply src/App.tsx]: thank you!

**Comment by sugarshin** [comment]
@cola119 I will merge this, this afternoon, could you please review if you have time. thank you!

**Comment by sugarshin** [comment]
thank you all!

**Pull request merged**